packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -42,7 +48,7 @@ const COPY_ICON_CLASS = 'elyra-copy-icon';
 const INSERT_ICON_CLASS = 'elyra-add-icon';
 
 /**
- * CodeSnippetTable props.
+ * CodeSnippetDisplay props.
  */
 interface ICodeSnippetProps {
```

Comment:
We prob want to rename this `I CodeSnippetDisplayProps` to match the class it's props for